### PR TITLE
✨ Add link button

### PR DESCRIPTION
**Pull request by thaoms**
Co-authored-by: Cedric <cedric.dhooge@engagor.com>

### CODE REVIEW

**Review by jensdevlieger — APPROVED** [review]
Cool cool cool

File: src/stories/LinkButton.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export const Danger = TemplateButton.bind({});
+
+export const Small = TemplateButton.bind({});
+
+Default.args = { mode: 'default' };
+Primary.args = { mode: 'primary' };
+Secondary.args = { mode: 'secondary' };
+Danger.args = { mode: 'danger' }
+
+Small.args = { mode: 'primary', size: 'small' };
```
Comment (llissssss) [inline src/stories/LinkButton.stories.tsx:30]: Q: what is this file for? for the storeybook? it is not used though, or is it magically imported?

**Review by llissssss — APPROVED** [review]
lgtm, but could you answer my question? :)

File: src/stories/LinkButton.stories.tsx (hunk 1 above)
Comment (thaoms) [reply src/stories/LinkButton.stories.tsx:30]: Storybook gathers these files and creates the storybook from it. See the configuration files in `.storybook`

Comment (thaoms) [reply src/stories/LinkButton.stories.tsx:30]: This one specifically
https://github.com/engagor/unified-components-react/blob/main/.storybook/main.js

**Pull request merged**